js/join.js

```diff
@@ -410,3 +449,133 @@ submit.addEventListener('click', async () => {
 copyBtn.addEventListener('click', () => {
   navigator.clipboard.writeText(`Real Dev Squad Verification Link: ${url}`);
 });
+
+document
+  .querySelector('#introduction')
+  .addEventListener('mouseleave', (event) => {
+    let count = 0;
+    let textArea = document.querySelector('#introduction').value;
+    let m;
+    const regex = /\w+/g;
+
+    while ((m = regex.exec(textArea)) !== null) {
+      // This is necessary to avoid infinite loops with zero-width matches
+      if (m.index === regex.lastIndex) {
+        regex.lastIndex++;
+      }
+
+      // The result can be accessed through the `m`-variable.
+      m.forEach((match, groupIndex) => {
+        count++;
+      });
+    }
+    if (count < 100) {
+      dataValidator(document.querySelector('#introduction'), 100 - count);
+    } else if (count >= 100) {
+      document.querySelector('#introductionCounter').innerText = '';
+      document
+        .querySelector('#introduction')
+        .classList.remove('incorrect-data');
+    }
+  });
+document.querySelector('#skills').addEventListener('mouseleave', (event) => {
+  let textArea = document.querySelector('#skills').value;
+  let count = 0;
+  let m;
+  const regex = /\w+/g;
+
+  while ((m = regex.exec(textArea)) !== null) {
+    // This is necessary to avoid infinite loops with zero-width matches
+    if (m.index === regex.lastIndex) {
+      regex.lastIndex++;
+    }
+
+    // The result can be accessed through the `m`-variable.
+    m.forEach((match, groupIndex) => {
+      count++;
+    });
+  }
+
+  if (count < 5) {
+    dataValidator(document.querySelector('#skills'), 5 - count);
+  } else if (count >= 5) {
+    document.querySelector('#skillsCounter').innerText = '';
```

Comment:
For this type of statement please create variables the use it

Reply:
Done